server/services/courseService.test.ts

```diff
@@ -19,6 +20,46 @@ describe('CourseService', () => {
     courseClientBuilder.mockReturnValue(courseClient)
   })
 
+  describe('createParticipation', () => {
+    const person = personFactory.build()
+
+    it('creates a course participation', async () => {
```

Comment:
I wonder if it's worth putting this in a `describe` block too? I'm not sure whether we should treat having a real `courseId` as more default than an `otherCourseName`? Perhaps it is